### ISSUE TITLE
Missing documentation: Configure DocumentServer to run on same server as nextcloud

### ISSUE THREAD

**Issue by DreamFlasher** [ONLYOFFICE/DocumentServer#369]
Main goal: Run DocumentServer and Nextcloud on same machine
Main problem: DocumentServer uses ports 80 and 443, which are already in use by nextcloud

Sub problems:
1) There are multiple locations of configuration files for DocumentServer and no documentation which ones are actually used. Eg. `/etc/onlyoffice/documentserver`, `/etc/nginx/conf.d`, `/etc/nginx/includes/`
2) The docker image is not a solution as it requires manually copying ssl certificates on a regular basis (let's encrypt requires frequent renewal)
3) Missing documentation on how to modify which config file such that non-default ports for http and https are used (nextcloud requires ssl)

**Comment by ShockwaveNN** [comment]
https://helpcenter.onlyoffice.com/server/linux/document/linux-installation.aspx
paragraph `Changing the default Document Server port`?

**Comment by DreamFlasher** [comment]
@ShockwaveNN I am aware of this documentation, it adds yet another way of configuring onlyoffice, which also does not work. Also this is only about the http port, which is the least relevant one -- this is mostly about https, see above.

**Comment by ShockwaveNN** [comment]
@DreamFlasher This one?
https://helpcenter.onlyoffice.com/server/linux/document/switch-to-https.aspx
(please note currently there is typo and you should use `for CentOS-based distributions:` to find `onlyoffice-documentserver-ssl.conf.template`. I fixed in https://github.com/ONLYOFFICE/helpcenter/pull/8 (private repo)

**Comment by DreamFlasher** [comment]
@ShockwaveNN No, this is about switching to https. Not about changing the https port. I have been working on solving this for a couple of days now, so I am pretty certain that my statement in the above report is valid, that there is no documentation for that. The closest I found was: https://help.nextcloud.com/t/nextcloud-13-and-onlyoffice-on-the-same-machine-without-docker-its-working/31721 -- so it seems to be possible, but I find a bunch of people saying that they never managed to get the documentserver running without docker. documentserver seems to be a configuration hell, so really a good start would be to know: What are all the configuration files and other hacks that influence the configuration of documentserver?
